06-building-a-program-start-to-finish/solving-the-problem/output/app/class/invertedIndexClass.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * Inverted Index class
+ * @class
+ */
+class InvertedIndex {
+
+  /**
+   * class constructor
+   * @constructor
+  */
+  constructor() {
+    this.indices = {};
+    this.indexedFiles = {};
+  }
+
+  readFile(fileToRead) {
```

Comment:
Where is the JSDoc for this function?

Not sure "fileToRead" is an appropriate parameter name. Please address.